streams/src/main/java/org/apache/kafka/streams/processor/internals/StampedRecord.java

```diff
@@ -49,6 +49,14 @@ public Headers headers() {
         return value.headers();
     }
 
+    public int serializedKeySize() {
```

Comment:
Please see my other comment above: how about just have a `sizeInBytes` function which takes key/value/timestamp/offset/topic/partition/headers into account?

Reply:
These functions seem not used any more?